backend/src/apiserver/config/sample_config.json

```diff
@@ -5,9 +5,14 @@
     "file":"/samples/xgboost-spark/xgboost-training-cm.py.tar.gz"
   },
   {
-    "name":"[Sample] ML - TFX - Taxi Tip Prediction Model Trainer",
-    "description":"Example pipeline that does classification with model analysis based on a public tax cab BigQuery dataset. For source code, refer to https://github.com/kubeflow/pipelines/tree/master/samples/tfx",
-    "file":"/samples/tfx/taxi-cab-classification-pipeline.py.tar.gz"
+    "name":"[Sample] ML - TFX - Taxi Tip Prediction Model Trainer on GCP",
+    "description":"Example pipeline that does classification with model analysis based on a public tax cab BigQuery dataset on GCP. For source code, refer to https://github.com/kubeflow/pipelines/tree/master/samples/tfx",
+    "file":"/samples/tfx/taxi-cab-classification-pipeline-gcp.py.tar.gz"
+  },
+  {
+    "name":"[Sample] ML - TFX - Taxi Tip Prediction Model Trainer for on-prem cluster",
```

Comment:
Will this sample run out of the box on GKE deployment?

Reply:
We would like to have the OOB sample for Kubeflow Pipelines. Do you think we should just put one of them(GCP)? if so, I will remove the onprem sample from OOB, thanks.

Reply:
@IronPan @paveldournov @gaoning777 @hongye-sun  WDYT?

Reply:
@Ark-kun Reconsider this, we just show one of them is enough, and with the last solution, need to compile for onprem cluster, will remove this and only keep OOB sample for GCP. Thanks,